app/src/main/java/de/westnordost/streetcomplete/quests/police_type/PoliceType.kt

```diff
@@ -0,0 +1,21 @@
+package de.westnordost.streetcomplete.quests.police_type
+
+enum class PoliceType(val policeName: String, val operator: PoliceOperator) {
+    CARABINIERI("Arma dei Carabinieri", PoliceOperator.ARMA_DEI_CARABINIERI),
+    GUARDIA_COSTIERA("Guardia Costiera", PoliceOperator.GUARDIA_COSTIERA),
+    GUARDIA_DI_FINANZA("Guardia di Finanza", PoliceOperator.GUARDIA_DI_FINANZA),
+    POLIZIA_DI_STATO("Polizia di Stato", PoliceOperator.POLIZIA_DI_STATO),
+    POLIZIA_MUNICIPALE("Polizia Municipale", PoliceOperator.POLIZIA_MUNICIPALE),
+    POLIZIA_LOCALE("Polizia Locale", PoliceOperator.POLIZIA_LOCALE),
+    POLIZIA_STRADALE("Polizia Stradale", PoliceOperator.POLIZIA_DI_STATO),
+    POLIZIA_FERROVIARIA("Polizia Ferroviaria", PoliceOperator.POLIZIA_DI_STATO)
+}
+
+enum class PoliceOperator(val operatorName: String, val wikidata: String, val wikipedia: String) {
+    ARMA_DEI_CARABINIERI("Arma dei Carabinieri", "Q54852", "en:Carabinieri"),
```

Comment:
Hello. Is English Wikipedia linked for a specific reason? It has differences with respect to the Italian article, but I did not read through them, and cannot say which article is more comprehensive or "better".

Reply:
Thanks for reviewing!
I don't know why, but the ID presets (https://nsi.guide) uses the English Wikipedia, I'd love to change it to the Italian one as it is the one I use for the other operators, but I used the one I think is used more (since lots of people map with ID). Should I use the Italian Wikipedia? From a first impression I find the Italian one more complete. 🤔

Reply:
In Poland norm is to use Polish Wikipedia if possible.

Note that iD presets are not something free from mistakes - if Italian community prefers Italian links I can change it (I have commit rights to NSI repo, so I can edit directly and I will be able to find it).

And Italian police will be present in Italy, so it is not necessary to consult more widely.

Reply:
Yes, it would be much appreciated, thank you. It seems [a cleanup commit](https://github.com/osmlab/name-suggestion-index/commit/b322cbe96eeb47bca3bb56debd20ad0a51c197e7) from Dec 2020 replaced `it:Arma dei Carabinieri` with `en:Carabinieri`.

Reply:
By all means, use the Italian one. For some reason I changed it when I was doing https://github.com/osmlab/name-suggestion-index/pull/4752, and the Italian one is most certainly better. (Maybe when I was looking up Wikidata? I don't remember.) Sorry about that.

Reply:
It seems that only https://github.com/osmlab/name-suggestion-index/blob/main/data/operators/amenity/police.json#L21 needs change to `it:`

Is there any other change required? Such as adding Wikipedia linking where it is missing in NSI?

Reply:
@matkoniecz The Italian page name is slightly different. Need to change to `it:Arma dei Carabinieri` to get to [here](https://it.wikipedia.org/wiki/Arma_dei_Carabinieri). I thought (perhaps mistaken) that it didn't matter what Wikipedia language you use in the tag. Whatever consumer of the data _could_ use the sister pages to show you the one in your language, if available. (That doesn't mean that consumers of the data _do_ do that.)

Please feel free to add missing tags to NSI.

Reply:
Sure I will after this one will be merged, @dieterdreist suggested me in the list to remove the Wikipedia link and just use Wikidata, so I'll do a PR with all those changes, I'll also remove Corpo Forestale dello Stato which doesn't exist anymore.

Reply:
Sounds good, so: delete wikipedia link, keep wikidata link.